docs-json-parameters-table.html

```diff
@@ -94,9 +94,7 @@ <h5 class="narrow-title">Properties</h5>
           <template is="dom-repeat" items="[[model]]">
             <template is="dom-if" if="[[item.isArrayHeader]]">
               <section class="tr">
-                <div class="td param-type"></div>
```

Comment:
Removing  this "cells" is causing the layout of the table header to collapse. Please, fix this before change.

Reply:
@jarrodek ok, updated. Current mixins should affect unstyled state anyhow anymore. Only if they exist.